src/NuProj.Tasks/AssignTargetFramework.cs

```diff
@@ -46,6 +46,18 @@ private static ITaskItem ConvertToPackageFile(ITaskItem output)
             var packageDirectory = output.GetPackageDirectory();
             var targetFramework = frameworkNameMoniker.GetShortFrameworkName();
             var metadata = output.CloneCustomMetadata();
+
+            task.Log.LogMessage(
+                "ConvertToPackageFile: fileName={0} => targetFramework={3}, frameworkName={4}, frameworkProfile={5}, targetPath={1}, packageDirectory={2}", fileName,
+                targetPath, packageDirectory, targetFramework, frameworkNameMoniker.FullName, frameworkNameMoniker.Profile);
+
+            if (string.IsNullOrEmpty(targetFramework))
```

Comment:
Are we sure that this is always an error condition? Most folders support not having a target framework explicitly listed.